src/local-item.ts

```diff
@@ -8,7 +7,22 @@ type Item = {
   content: string;
 };
 
-const items = new Map<string, Item>();
+const items = new Map<any, any>();
```

Comment:
does this have to be any, any? Why can't it be string, Item?

Reply:
That would be the perfect solution. although, I have been blocked trying to make it work for a while, and since it's just a template project and it worked without specifying the return type, I put it like that in this PR. You basically get an error on the return type. I will take a look at this again and see if we can make it work